content/docs/next-release/spm.md

```diff
@@ -172,6 +172,93 @@ the HTTP API.
 
 ## Troubleshooting
 
+### Check the /metrics endpoint
+
+The `/metrics` endpoint can be used to check if spans for specific services were received.
+The `/metrics` endpoint is served from the admin port.
+Assuming that Jaeger all-in-one and query are available under hosts named `all-in-one`
+and `jaeger-query` respectively, here are sample `curl` calls to obtain the metrics:
+
+```shell
+$ curl http://all-in-one:14269/metrics
+
+$ curl http://jaeger-query:16687/metrics
+```
+
+The following metrics are of most interest:
+
+```shell
+# all-in-one
+jaeger_requests_total
+jaeger_latency_bucket
+
+# jaeger-query
+jaeger_query_requests_total
+jaeger_query_latency_bucket
+```
+
+Each of these metrics will have a label for each of the following operations:
+```shell
+get_call_rates
+get_error_rates
+get_latencies
+get_min_step_duration
+```
+
+If things are working as expected, the metrics with label `result="ok"` should
+be incrementing, and `result="err"` being static. For example:
+```shell
+jaeger_query_requests_total{operation="get_call_rates",result="ok"} 18
+jaeger_query_requests_total{operation="get_error_rates",result="ok"} 18
+jaeger_query_requests_total{operation="get_latencies",result="ok"} 36
+
+jaeger_query_latency_bucket{operation="get_call_rates",result="ok",le="0.005"} 5
+jaeger_query_latency_bucket{operation="get_call_rates",result="ok",le="0.01"} 13
+jaeger_query_latency_bucket{operation="get_call_rates",result="ok",le="0.025"} 18
+
+jaeger_query_latency_bucket{operation="get_error_rates",result="ok",le="0.005"} 7
+jaeger_query_latency_bucket{operation="get_error_rates",result="ok",le="0.01"} 13
+jaeger_query_latency_bucket{operation="get_error_rates",result="ok",le="0.025"} 18
+
+jaeger_query_latency_bucket{operation="get_latencies",result="ok",le="0.005"} 7
+jaeger_query_latency_bucket{operation="get_latencies",result="ok",le="0.01"} 25
+jaeger_query_latency_bucket{operation="get_latencies",result="ok",le="0.025"} 36
+```
+
+If there are issues reading metrics from Prometheus such as a failure to reach
+the Prometheus server, then the `result="err"` metrics will be incremented. For example:
+```shell
+jaeger_query_requests_total{operation="get_call_rates",result="err"} 4
+jaeger_query_requests_total{operation="get_error_rates",result="err"} 4
+jaeger_query_requests_total{operation="get_latencies",result="err"} 8
+```
+
+At this point, checking the logs will provide more insight towards root causing
+the problem.
+
+### Query Prometheus
+
+Graphs may still appear empty even when the above jaeger metrics indicate successful reads
```

Comment:
```suggestion
Graphs may still appear empty even when the above Jaeger metrics indicate successful reads
```

I this because the reads for non-existing metrics still succeed in Prometheus? Is it possible to detect that elsewhere, e.g. by emitting another metric that counts the number of data points returned from Prom?

Reply:
> I (sic) this because the reads for non-existing metrics still succeed in Prometheus? 

Yup: `curl http://localhost:9090/api/v1/query?query=doesnotexist` will return HTTP status code `200` and an empty result.

> Is it possible to detect that elsewhere, e.g. by emitting another metric that counts the number of data points returned from Prom?

Ah, so perhaps this is what the [`response` counter](https://github.com/jaegertracing/jaeger/pull/3667#discussion_r868088359) was for, to build up a histogram on the number of results returned.

I'll create a follow-up PR for that.